libs/ui/src/lib/terminal/terminal.component.ts

```diff
@@ -40,15 +40,21 @@ export class TerminalComponent implements AfterViewInit, OnDestroy {
   @Input() command: string;
 
   @Input()
-  set input(s: string) {
+  set inputStream(s: string) {
```

Comment:
I know what you are trying to indicate here, but I think the stream suffix can be misleading. Folks often refer to Observables as streams. That's why I thought when I looked it it. 

We use the suffix chunk on the backend. Maybe it's better?

We can also change it around and actually pass an observable here and subscribe to it here. Currently we use |async pipe everywhere to get the input from the observable.

Reply:
I changed to out / outChunk so that the naming conventions used on the server and frontend are the same